doc/source/exclude_patterns.md

```diff
@@ -0,0 +1,93 @@
+# Exclude Pattern Settings 
+- Optional argument for [in-toto-run.](https://in-toto.readthedocs.io/en/latest/command-line-tools/in-toto-run.html) Path patterns to match paths that should not be recorded as 'materials' or 'products'.  Passed patterns override patterns defined in environment variables or config files (see ```ARTIFACT_EXCLUDE_PATTERNS``` documentation for additional info) 
+- Uses [PathSpec](http://python-path-specification.readthedocs.io) to compile [gitignore-style patterns](https://git-scm.com/docs/gitignore), implementing GitWildMatchPattern class which itself is derived from [Rsync's wildmatch](https://github.com/AndyA/rsync/blob/master/lib/wildmatch.c)   
```

Comment:
I think this should be clarified. Do you mean git's patterns are derived from rsync's wildmatch?

Reply:
Yep! I think that we might not need that detail since we're focusing on and linking the Pathspec documentation, which already does a fantastic job describing how Git's patterns are derived from RSync's wild match.